backends/bmv2/common/control.cpp

```diff
@@ -412,6 +415,7 @@ ControlConverter::convertTable(const CFG::TableNode* node,
             tkey->append(keyelement);
         }
     }
+    LOG1("table_match_type: " << table_match_type);
```

Comment:
isn't `LOG1` for the most "important" log messages?

Reply:
Yes, I thought, to debug any future issue in the method, we could use a LOG1 and leave other logging clutter out.

However, I am open to change.

Would you like me to change LOG1 to LOG3?

Reply:
It's a pretty minor piece of information that is also available in the generated JSON.
It has to be at least a LOG3, since the name of the table itself is logged with that verbosity:
```
LOG3("Processing " << dbp(table));
```